frontend/src/metabase/dashboard/reducers.js

```diff
@@ -317,6 +317,9 @@ const draftParameterValues = handleActions(
         return state;
       },
     },
+    [FETCH_DASHBOARD]: {
```

Comment:
nit: Should we reset it to an empty object instead?

Reply:
Actually, we should only update `draftParameterValues` when `dashboard.auto_apply_filters` is `false` since we expect it to be empty when `auto_apply_filters` is `true`